### PR TITLE
fix: eth_getlogs not including txhash, blockhash

### PR DESCRIPTION
**Pull request by jorgemmsilva**

### CODE REVIEW

File: packages/vm/core/evm/emulator/statedb.go [hunk 1]
```diff
@@ -56,6 +56,9 @@ type StateDB struct {
 	getBalance GetBalanceFunc
 	subBalance SubBalanceFunc
 	addBalance AddBalanceFunc
+
+	thash   common.Hash
+	txIndex uint
```
Comment (dessaya) [inline packages/vm/core/evm/emulator/statedb.go]: These are unused?

Also (unrelated to this PR): `StateDB` documentation seems outdated:
```
// The Ethereum account balance is tied to the L1 balance, and immutable
// from EVM. (Instead, funds are supposed to be manipulated via the ISC
// sandbox).
```

File: packages/vm/core/evm/evmtest/evm_test.go [hunk 2]
```diff
@@ -990,3 +994,6 @@ func TestSendEntireBalance(t *testing.T) {
 	env.soloChain.AssertL2BaseTokens(isc.NewEthereumAddressAgentID(ethAddr), 0)
 	env.soloChain.AssertL2BaseTokens(someEthereumAgentID, currentBalance-tokensForGasBudget)
 }
+
+func TestEVMLogs(t *testing.T) {
+}
```
Comment (dessaya) [inline packages/vm/core/evm/evmtest/evm_test.go]: empty test?

**Review by dessaya — APPROVED** [review]

File: packages/vm/core/evm/emulator/statedb.go (hunk 1 above)
Comment (jorgemmsilva) [reply packages/vm/core/evm/emulator/statedb.go]: yeah, the linter agrees ahah.
Its not running on my machine for some reason :shrug:

Comment (jorgemmsilva) [reply packages/vm/core/evm/emulator/statedb.go]: I'll update the docs, good catch :+1:

File: packages/vm/core/evm/evmtest/evm_test.go (hunk 2 above)
Comment (jorgemmsilva) [reply packages/vm/core/evm/evmtest/evm_test.go]: I heard more tests = good :trollface:

**Pull request merged**